### PR TITLE
v4 collapse without jquery

### PR DESCRIPTION
**Pull request by Johann-S**
~Work in progress...~

### CODE REVIEW

File: js/src/collapse.js [hunk 1]
```diff
@@ -203,36 +203,35 @@ const Collapse = (() => {
...
       Util.reflow(this._element)
 
-      $(this._element)
-        .addClass(ClassName.COLLAPSING)
-        .removeClass(ClassName.COLLAPSE)
-        .removeClass(ClassName.SHOW)
+
+      this._element.classList.add(ClassName.COLLAPSING)
+      this._element.classList.remove(ClassName.COLLAPSE)
+      this._element.classList.remove(ClassName.SHOW)
```
Comment (XhmikosR) [inline js/src/collapse.js]: You can comma separate the selectors. Same goes for `add` BTW.

Comment (Johann-S) [reply js/src/collapse.js]: they are no support in IE10 for that : https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Browser_compatibility

Comment (XhmikosR) [reply js/src/collapse.js]: Ah, all right. Freaking IE.

**Comment by Johann-S** [comment]
Done 👍 waiting for https://github.com/twbs/bootstrap/pull/23989 before I can merge this one
if you have time @XhmikosR do not hesitate to take a look at this PR, thanks 👍

**Pull request merged**